nuxt.config.js

```diff
@@ -62,158 +62,7 @@ module.exports = {
-          file: 'cs.js',
-          iso: 'cs-CZ'
-        },
-        {
-          name: 'Dansk',
-          code: 'da',
-          file: 'da.js',
-          iso: 'da-DK'
-        },
-        {
-          name: 'Deutsch',
-          code: 'de',
-          file: 'de.js',
-          iso: 'de-DE'
-        },
-        {
-          name: 'Ελληνικά',
-          code: 'el',
-          file: 'el.js',
-          iso: 'el-GR'
-        },
-        {
-          name: 'English',
-          code: 'en',
-          file: 'en.js',
-          iso: 'en-GB'
-        },
-        {
-          name: 'Español',
-          code: 'es',
-          file: 'es.js',
-          iso: 'es-ES'
-        },
-        {
-          name: 'Eesti',
-          code: 'et',
-          file: 'et.js',
-          iso: 'et-EE'
-        },
-        {
-          name: 'Euskara',
-          code: 'eu',
-          file: 'eu.js',
-          iso: 'eu-ES'
-        },
-        {
-          name: 'Suomi',
-          code: 'fi',
-          file: 'fi.js',
-          iso: 'fi-FI'
-        },
-        {
-          name: 'Français',
-          code: 'fr',
-          file: 'fr.js',
-          iso: 'fr-FR'
-        },
-        {
-          name: 'Gaeilge',
-          code: 'ga',
-          file: 'ga.js',
-          iso: 'ga-IE'
-        },
-        {
-          name: 'Hrvatski',
-          code: 'hr',
-          file: 'hr.js',
-          iso: 'hr-HR'
-        },
-        {
-          name: 'Magyar',
-          code: 'hu',
-          file: 'hu.js',
-          iso: 'hu-HU'
-        },
-        {
-          name: 'Italiano',
-          code: 'it',
-          file: 'it.js',
-          iso: 'it-IT'
-        },
-        {
-          name: 'Lietuvių',
-          code: 'lt',
-          file: 'lt.js',
-          iso: 'lt-LT'
-        },
-        {
-          name: 'Latviešu',
-          code: 'lv',
-          file: 'lv.js',
-          iso: 'lv-LV'
-        },
-        {
-          name: 'Malti',
-          code: 'mt',
-          file: 'mt.js',
-          iso: 'mt-MT'
-        },
-        {
-          name: 'Nederlands',
-          code: 'nl',
-          file: 'nl.js',
-          iso: 'nl-NL'
-        },
-        {
-          name: 'Polski',
-          code: 'pl',
-          file: 'pl.js',
-          iso: 'pl-PL'
-        },
-        {
-          name: 'Português',
-          code: 'pt',
-          file: 'pt.js',
-          iso: 'pt-PT'
-        },
-        {
-          name: 'Română',
-          code: 'ro',
-          file: 'ro.js',
-          iso: 'ro-RO'
-        },
-        {
-          name: 'Slovenčina',
-          code: 'sk',
-          file: 'sk.js',
-          iso: 'sk-SK'
-        },
-        {
-          name: 'Slovenščina',
-          code: 'sl',
-          file: 'sl.js',
-          iso: 'sl-SI'
-        },
-        {
-          name: 'Svenska',
-          code: 'sv',
-          file: 'sv.js',
-          iso: 'sv-SE'
-        }
-      ],
+      locales: require('./plugins/i18n/locales.json'),
```

Comment:
would it be neater if you created a locales const at the top of the page?

Reply:
Indeed, and done.